src/js/components/Sidebar/Sidebar.js

```diff
@@ -1,13 +1,22 @@
 import React from 'react';
 import { Box } from '../Box';
 import { SidebarPropTypes } from './propTypes';
+import { useThemeValue } from '../../utils/useThemeValue';
 
-export const Sidebar = ({ children, footer, header, ...rest }) => (
-  <Box pad="small" gap="large" height={{ min: '100%' }} {...rest}>
-    {header}
-    <Box flex>{children}</Box>
-    {footer}
-  </Box>
-);
+export const Sidebar = ({ children, footer, header, ...rest }) => {
+  const { theme } = useThemeValue();
+  return (
+    <Box
+      height={{ min: '100%' }}
+      gap={theme.sideBar.gap}
```

Comment:
```suggestion
      gap={theme.sidebar.gap}
```
since component is "Sidebar", should we keep same casing (not capitalize B)